### DISCUSSION TITLE
Вопросы и предложения

### DISCUSSION THREAD

**Discussion by pingvin1308** [pingvin1308/pingvin1308#3] [Q&A]
Сюда можно написать ваш вопрос или предложение (КЭП)

За вопрос или предложение можно проголосовать стрелочками вверх или вниз.
Таким образом будет понятно что и кому интересно :)

**Comment by pingvin1308** [comment]
Интересно было бы послушать о последний достижениях проекта OCitizens.
- Сколько людей в работе или в планах
- Интересные кейсы с которыми столкнулись недавно

**Comment by pingvin1308** [comment]
Куда податься в нынешних условиях) Особенно семейным

Оригинал: ![image](https://user-images.githubusercontent.com/17434920/194496900-08757d25-37bb-427d-8b35-872c7d3a12a2.png)
https://t.me/c/1744734197/58

**Comment by pingvin1308** [comment]
Как собирать доказательства на О1 в условиях вынужденного переезда?

Оригинал:
![image](https://user-images.githubusercontent.com/17434920/194497068-4fd3029a-706e-4f9a-b4dd-49cddfae78c7.png)
![image](https://user-images.githubusercontent.com/17434920/194497095-983d19c4-ed62-471e-b292-9c0319e16f2e.png)

**Comment by dmitriyzima** [comment]
Есть ли возможность провести онлайн встречу с проверенным иммиграционным адвокатом по распространенным вопросам? в частности по визам отличным от О1

**Comment by KatherineKiseleva** [comment]
Когда начинали вести youtube, монетизации не было, а сил и времени уходило много.
Как себя мотивировали продолжать снимать, монтировать, искать людей для интервью?

Что помогало не останавливаться, а продолжать работать?

**Comment by Aleksei-Iagur** [comment]
Чем отличается поиск работы в России и зарубежом?
Каков наилучший способ/план подготовки к собеседованиям в настоящее время?

**Comment by AlexeyVykhrystyuk-Strasz** [comment]
А вот кстати интересно про ваш маршрут узнать и как вы к нему пришли. Если это публичная информация)

Оригинал:
![image](https://user-images.githubusercontent.com/72028358/194720459-90b05757-7e5f-46de-9d44-2ee4b826facc.png)
https://t.me/c/1744734197/99

**Comment by AVykhrystyuk** [comment]
Ответ, скорее всего, уже был дан ранее на канале, но хотелось бы услышать ваш "up-to-dated" личный список из топ 5-10 плюсов и минусов жизни в Америке?

**Comment by balmasov** [comment]
В последнее время чаще говорят про кризис в IT сфере в странах европы и США, что думаете на этот счет? Повлияет ли это на поиск работы в США?

**Comment by balmasov** [comment]
Что проше получить визу 01 или Green Card, если есть жена с гражданством США? :))

**Comment by balmasov** [comment]
Есть ли в США вариант устройства или заключения трудового договора по ИП (Грузинское)? То есть если есть Грузинское ИП, то можно ли заключить с фирмой договор на оказание им услуг, при условии, что есть гринкарта или разрешение на рбаоту в США?

**Comment by maxuver** [comment]
Планы действий для получения визы help, гуманитарной визы, визы переселенца.

Подводные камни, ошибки и что не стоит делать при подаче на гуманитарную визу. 

Если есть документ доказывающий украинское происхождение(но сам гражданин РФ), то могу ли подавать на CUAET и переезд в Канаду или другие страны? Возможно из Грузии подать на CUAET через обращение в украинское посольство?

Что ещё возможно сделать, чтобы получить помощь?

**Comment by maxuver** [comment]
Если по профессии айтишник, есть опыт работы и апостиль диплома вуза, то на какие иностранные программы можно обратиться в содействии при поиске работы? Как получить пособие на время поиска работы, если это возможно?

Интересуют: страны ЕС, Грузия, Турция, США, Канада

P.S. Инкубаторы для айтишников может бывают?)

**Comment by komandakycto** [comment]
### Про миграцию

- Топ три страны сейчас для людей из бывшего СНГ где можно легализоваться и работать.
- Лучшая страна для семейного проживания.
- Нужны ли ИТшники и около того сейчас в США? Мы много слышали как топ компании сокращают найм, инфляция, мировой кризис.
- Как понять свои шансы с визой О1 и с чего начать?
- Герои ваших роликов это успешные исключения из правил? Каков процент провалов с визой О1?
- На сколько компании США готовы брать на себя оформление визы сотрудника и его семьи чтобы привезти его? В каком случае они это делают? Примеры таких компаний?
- Топ три альтернативы кремниевой долины В США и в мире?
- А надо ли ехать в США и долину в частности?
- Новая Зеландия и Южная Америка недооценены?
- Как не сойти с ума в условиях вынужденной миграции? Как поддерживать психологическую устойчивость?
- Как разговаривать с близкими родственниками и родителями на тему миграции? Как объяснить свое желание переехать?
- Как развивать карьеру в миграции?
- Как вести финансы в условиях эмиграции? Биржы недоступны, банковские карты заблокированы, необанки не работают с РФ, Украиной и Белоруссией.
- Медицина в эмиграции, страховки и лайвхаки?

### Про личное
- Почему уехали из США? Планируете ли вернуться?
- Какого это жить в стране где легализовано оружие, легкие наркотики, а на улицах много бездомных? Чувствовали ли вы себя в безопасности?
- Может ли рядовой разработчик (Senior) жить не в 0 в долине?
- Есть ли в США доступное и хорошее школьное (не университет) образование?
- Какие страны планируете посетить? И почему именно их?
- Где бы вы хотели давать образование своему ребенку/детям?
- Тоска по родине проходит? Или это с нами навсегда?
- Вы бывали на родине за последние 6 лет? Хотелось бы, если нет?
- Что помогло не сломаться в периоды депрессии в США?
- Топ три человека у кого бы вы хотели взять интервью?
- Планы на будущее?

**Comment by alvteren** [comment]
Какие документы могут понадобиться если в будущем апрувнут O-1, которые нельзя сделать за пределами России? Вобщем что обязательно нужно сделать перед отъездом из России

**Comment by mkrup** [comment]
Если принять участие в лотерее гринкарт, то закрывает ли это путь к другим американским визам, в том числе к туристической (слышал, что это считается попыткой иммиграции и туристическую могу не дать)?

**Reply by ZhukMax** [reply]
пока не выиграл и не явился на собеседование это не считается миграционным намерением

**Comment by dc-international** [comment]
Как UI-UX дизайнеру/арт-директору с 10-летним опытом работы и семьей с 2 детьми устроиться на работу дизайнером в американскую компанию и переехать в США?

**Comment by telnoiko** [comment]
Важно ли получение ВНЖ или разрешения на проживание в стране при собеседовании в консульстве? Слышал что нужно показывать что ты путешествуешь с таких случаях. Сейчас многие выехали из России и есть возможность просто жить в разных/одной странах либо же целенаправленно получать ВНЖ в какой-то конкретной стране.

**Comment by vsalex** [comment]
Интересно послушать про менталитет американцев, какая специфика работы / адаптации в американской компании. На что обращать внимание в коммуникациях, на какие грабли натыкались?

**Comment by kamax70** [comment]
Кремневая долина или Сиетл? Как у штата Вашингтон плюсы?

**Comment by pingvin1308** [comment]
Вопрос возможно покажется групым, но все же. Я действительно не знаю этого :)
Как происходит процедура обновления загранпаспорта в Армерике?
Или можно забить на загранпаспорт при наличии грин карты/миграционной визы?
